content/sensu-go/6.8/catalog/catalog-reference.md

```diff
@@ -0,0 +1,1229 @@
+---
+title: "Catalog integrations reference"
+linkTitle: "Catalog Integrations Reference"
+reference_title: "Catalog integrations"
+type: "reference"
+description: "Use Sensu's Catalog API to build a private catalog of Sensu integrations. Read the reference to create integrations for your catalog."
+weight: 80
+version: "6.8"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.8:
+    parent: catalog
+---
+
+{{% notice commercial %}}
+**COMMERCIAL FEATURE**: Access the Sensu Catalog and integrations in the packaged Sensu Go distribution.
+For more information, read [Get started with commercial features](../../commercial/).
+{{% /notice %}}
+
+{{% notice note %}}
+**NOTE**: The Sensu Catalog is in public preview and is subject to change.
+{{% /notice %}}
+
+The [Sensu Catalog][1] is a collection of Sensu integrations that provide reference implementations for effective observability.
+The contents of the official Sensu Catalog are periodically published to the official Sensu Catalog API, which is hosted at https://catalog.sensu.io and displayed within the Sensu web UI.
+
+When users install integrations in the Sensu web UI, they receive prompts to enter information.
+For example, the DNS Monitoring integration includes prompts for the domain name, record type, record class, servers, and port to query.
+Sensu then applies the user's customizations to the integration's resource definitions and automatically deploys the integration configuration to agents in real time.
+No external configuration management is required.
+
+## Catalog repository example
+
+The repository that stores Sensu integrations must organize files in the following structure:
+
+{{< code text >}}
+integrations/
+└── <namespace>/
+    └── <integration_name>/
+        ├── img/
+        │   ├── dashboard-1.gif
+        │   └── dashboard-2.png
+        ├── CHANGELOG.md
+        ├── README.md
+        ├── logo.png
+        ├── sensu-integration.yaml
+        └── sensu-resources.yaml
+    └── <integration_name>/
+        ├── img/
+        │   ├── dashboard-1.gif
+        │   └── dashboard-2.png
+        ├── CHANGELOG.md
+        ├── README.md
+        ├── logo.png
+        ├── sensu-integration.yaml
+        └── sensu-resources.yaml
+└── <namespace>/
+    └── <integration_name>/
+        ├── img/
+        │   ├── dashboard-1.gif
+        │   └── dashboard-2.png
+        ├── CHANGELOG.md
+        ├── README.md
+        ├── logo.png
+        ├── sensu-integration.yaml
+        └── sensu-resources.yaml
+{{< /code >}}
```

Comment:
I think this section should include a note that explains that in the context of catalog integration organization, namespace does not mean the Sensu RBAC namesapce.